src/poliastro/twobody/orbit.py

```diff
@@ -839,6 +852,53 @@ def frozen(
                 )
             )
 
+    def calculate_pass(self, lat, long, H):
+        a = self.a  # semimajor axis
+        e = self.ecc  # eccentricity
+        i = self.inc  # equatorial inclination
+        O = self.long_asc  # longitude of ascending node
+        o = self.argp  # argument of perigee
+        T = self.period  # time of perifocal passage
+
+        n = self.n  # mean motion
+        ar = self.attractor.r  # equatorial radius
+        f = 0  # TODO: flattening
+
+        t0 = 1  # TODO: epoch universal time (rad)
```

Comment:
I propose we set this reference time to the corresponding "time of last perifocal passage" of the of the orbit we're studying. In that case, in (18) the `T` cancels out (we don't even need it here in the code) and we're left with calculating the Greenwich Mean Sidereal Time at that particular instant.